### DISCUSSION TITLE
Publish Zigbee end devices firmware version

### DISCUSSION THREAD

**Discussion by andreamoretto** [Koenkk/zigbee2mqtt#12842] [Devices]
Hi,

  it would be nice to display the Zigbee devices firmware versions. As far as I know the firmware version is not displayed, is there any way to query the firmware versions?

Thanks in advance

**Comment by Nardol** [comment]
In [MQTT setting](https://www.zigbee2mqtt.io/guide/configuration/mqtt.html#server-connection), if you set `include_device_information: true` you will have device information included in MQTT messages.
According to the documentation, default value is `false` for this setting.
When I used it, I was able to get firmware version.

**Reply by andreamoretto** [reply]
It worked, thanks very much!
